README.md

```diff
@@ -16,11 +19,18 @@ npm run build
 ```
 
 ## Deployment
-Due to `.app` now being a publicly available TLD, we suggest developing with a `.test` TLD.
+We suggest developing with a `.test` TLD.
+
+1. `npm run build`
+2. tar -cvzf ARCVMarket_v<x.y.z>(-[beta|RC]X) ./dist
```

Comment:
Backticks here, and around everything in step 5?

Reply:
added!